packages/auth/src/index.ts

```diff
@@ -230,9 +230,7 @@ export const start = async (
       logger.info('completed graceful shutdown.')
       process.exit(0)
     } catch (err) {
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-ignore
-      const errInfo = err instanceof Error && error.stack ? err.stack : err
```

Comment:
This case illustrates why we should prefer type assertions over `ts-ignore` comments. This `ts-ignore` was inadvertently hiding a `ReferenceError`. We call `error.stack` but error is not defined anywhere.